ddmrp/data/ir_cron.xml

```diff
@@ -10,9 +10,8 @@
         <field name="numbercall">-1</field>
         <field name="priority">6</field>
         <field eval="False" name="doall"/>
```

Comment:
Please add `<field name="state">code</field>`

Reply:
Oops :sweat: I added them but removed them afterwards. Weird it's not the default value.
Fixed now